### PR TITLE
Add to project index.html, application.scss, .gitignore

### PR DESCRIPTION
**Pull request by jeymarsonwolox**
Summary
Creation of the project, configuration of sass and  gitignore.

Screenshots
captura de pantalla 2018-10-30 a la s 2 20 31 p m

Trello Card
https://trello.com/c/BRjY95n0/11-setup-del-proyecto

### CODE REVIEW

File: scss/styles.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.title {
```
Comment (FrankIglesias) [inline scss/styles.scss:1]: check indentation

**Review by FrankIglesias — CHANGES_REQUESTED** [review]
add empty line at the bottom of the file

**Comment by jeymarsonwolox** [comment]
Correction in file indentation

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (FrankIglesias) [inline .gitignore]: add empty line to the gitignore

**Review by FrankIglesias — CHANGES_REQUESTED** [review]
Just one fix and go

**Comment by jeymarsonwolox** [comment]
Empty line is added to gitignore

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**